client/src/components/EstatePage.css

```diff
@@ -143,3 +143,34 @@
   margin-top: 12px;
   font-size: smaller;
 }
+
+.propertyCard {
+  width: 345px;
+  margin: 0 auto;
+  margin-top: 25px;
+  box-shadow: 0px 0px 5px 0px rgb(85, 85, 85);
+  color: #707070;
+  font-weight: 100;
+  background-color: #dceff8;
+  padding-bottom: 1px;
+}
+
+.p {
+  margin: 5px;
+}
+
+.linkBox {
+  text-decoration: none !important;
+}
+
+.img {
+  width: 345px;
```

Comment:
I presume this is only temporarily :) fixed `width` and `height` are rarely good for responsiveness

Reply:
sure just temporarily. I do responsivness after. but the more I do the more I think its not a good idea :D